firefox-ios/Client/Frontend/Browser/Tabs/Views/TabPeekViewController.swift

```diff
@@ -108,6 +108,18 @@ final class TabPeekViewController: UIViewController,
                 return
             })
         }
+        if tabPeekState.showRemoveBookmark {
+            actions.append(UIAction(title: .TabPeekRemoveBookmark,
+                                    image: UIImage.templateImageNamed(StandardImageIdentifiers.Large.bookmarkFill),
+                                    identifier: nil) { [weak self] _ in
+                guard let self else { return }
+                let action = TabPeekAction(tabUUID: self.tabModel.tabUUID,
+                                           windowUUID: self.windowUUID,
+                                           actionType: TabPeekActionType.removeBookmark)
+                store.dispatchLegacy(action)
```

Comment:
Looks good, with the mention that I've heard @lmarceau saying that we should use `dispatch` instead of `dispatchLegacy` when sending actions.